courses/CONTRIBUTING.md

```diff
@@ -2,10 +2,17 @@ We realise that the initial content we created is just a starting point and our
 
 As a contributor, you represent that the content you submit is not plagiarised. By submitting the content, you (and, if applicable, your employer) are licensing the submitted content to LinkedIn and the open source community subject to the Creative Commons Attribution 4.0 International Public License.
 
-We suggest to open an issue first and seek advice for your changes before submitting a pull request.
-
 *Repository URL*: [https://github.com/linkedin/school-of-sre](https://github.com/linkedin/school-of-sre)
 
+### Contributing Guidelines
+Ensure that you adhere to the following guidelines:
+
+* Should be about principles and concepts that can be applied in any company or individual project. Do not focus on particular tools or tech stack(which usually change over time).
+* Adhere to the Code of Conduct.
```

Comment:
This will be removed/retained depending on whether we decide to add a code of conduct.

Reply:
lets add a code of conduct, link that and merge this?
Code of conduct should call for inclusiveness and fair language